extension/chrome/settings/modules/keyserver.ts

```diff
@@ -113,17 +113,19 @@ Catch.try(async () => {
     }));
 
     $('.action_remove_alias').click(Ui.event.prevent('double', async self => {
```

Comment:
I think this functionality no longer makes sense. If the user wants to remove an alias, they can do it in Gmail and it will get refreshed next time by pulling from sendAs API. So I would remove the button & this handler.

Reply:
Agree, removed